ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec-CUI-nonprod.json

```diff
@@ -15,7 +15,8 @@
           "caseworker-civil-admin",
           "RES-SOL-ONE-UNSPEC-PROFILE",
           "RES-SOL-ONE-SPEC-PROFILE",
-          "RES-SOL-TWO-SPEC-PROFILE"
+          "RES-SOL-TWO-SPEC-PROFILE",
```

Comment:
Why are we only giving Read access and not CRUD?
Are we not updating these fields ever?

Reply:
because these fields are updated by lr applicant and we dont need defendant to update them. for dashboard we only need to read them

Reply:
However found one field that should be write too. not this one though lol